### PR TITLE
(PUP-8314) add decorations to raise ArgumentError strings part 6

### PR DESCRIPTION
**Pull request by ericldelaney**

### CODE REVIEW

File: lib/puppet/util/ldap/manager.rb [hunk 1]
```diff
@@ -44,7 +44,8 @@ def create(name, attributes)
   # Open, yield, and close the connection.  Cannot be left
   # open, at this point.
   def connect
-    raise ArgumentError, "You must pass a block to #connect" unless block_given?
+    #TRANSLATORS '#connect' is a method name and and should not be translated, 'block' refers to a Ruby code block
```
Comment (melissa) [inline lib/puppet/util/ldap/manager.rb]: does this mean that `block` should not be translated, or are you just giving context for how `block` should be translated?

File: lib/puppet/util/ldap/manager.rb [hunk 2]
```diff
@@ -85,7 +86,8 @@ def dn(name)
 
   # Convert an ldap-style entry hash to a provider-style hash.
   def entry2provider(entry)
-    raise ArgumentError, "Could not get dn from ldap entry" unless entry["dn"]
+    #TRANSLATOR 'dn' refers to a 'distinguished name' in LDAP (Lightweight Directory Access Protocol)
```
Comment (melissa) [inline lib/puppet/util/ldap/manager.rb]: We probably should add `and should not be translated`

Comment (melissa) [reply lib/puppet/util/ldap/manager.rb]: also, `ldap` probably shouldn't be translated

File: lib/puppet/util/ldap/manager.rb (hunk 1 above)
Comment (ericldelaney) [reply lib/puppet/util/ldap/manager.rb]: I was giving context for the translation

**Comment by ericldelaney** [comment]
@melissa Updated

**Pull request merged**